source/_components/ubee.markdown

```diff
@@ -10,8 +10,6 @@ footer: true
 logo: ubee.png
 ha_category: Presence Detection
 ha_release: 0.89
-redirect_from:
- - /components/device_tracker.ubee/
```

Comment:
We should keep this if there used to be to be a page at that place.

Reply:
Added it back.